scripts/installers/make-docker

```diff
@@ -40,10 +42,19 @@ def _ensure_docker_is_available():
 
 
 def _make_build_context(build_context_dir, exe):
+    _update_exe_metadata(exe)
```

Comment:
We should probably update the metadata of the copied exe instead of the source exe to make sure we do not accidentally inject the metadata in an artifact we plan to use later on.

Reply:
Yeah, it was temporary commit to run it through the pipeline